src/cli/args.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+let app = require('commander')
+let path = require('path')
+let ipfsAPI = require('ipfs-api')
+
+let Compiler = require('./compiler')
+
+function addBuildCommand() {
+    app
+        .version('0.1.0')
+        .arguments('<cmd> [file]')
```

Comment:
These two lines above don't really belong in this function. Let's rename `args.js` to `app.js` and add a `createApp()` function that includes these two calls + the `--verbosity` option.